PYME/cluster/HTTPDataServer.py

```diff
@@ -679,6 +681,64 @@ def get_h5_part(self, path):
         except IOError:
             self.send_error(404, "File not found - %s, [%s]" % (self.path, path))
             return None
+
+    def get_tabular_part(self, path):
+        """
+
+        Parameters
+        ----------
+        path: str
+            path to an hdf or h5r file on the dataserver computer. Specific parts of the file can be accessed by
+            appending, e.g. /Metadata, for a resulting `path` of something like folder/test.h5r/Metadata. The 'part' of
+            the file returned defaults to FitResults if unspecified, and can additionally include a slice to take before
+            returning. In other words, a `path` of 'folder/test.h5r/FitResultsslice(0,100,None)' will return the first
```

Comment:
I really don't like just appending slice without any form of separator. I'd consider either of:

`test.h5r/FitResults/start/end`

or, potentially better, using standard query syntax:

`test.h5r/FitResults?from=0&to=100`

Reply:
We could potentially be really fancy and allow the caller to specify the file type they want:
e.g.:

`test.h5r/FitResults.json?from=0&to=100`
or 
`test.h5r/FitResults.npy?from=0&to=100`

This would be a good fit with keeping our web UI as language agnostic as possible whilst not forcing us to use json for everything.